Mathlib/Algebra/Group/Fin.lean

```diff
@@ -123,9 +123,55 @@ lemma sub_one_lt_iff {k : Fin (n + 1)} : k - 1 < k ↔ 0 < k :=
   not_iff_not.1 <| by simp only [lt_def, not_lt, val_fin_le, le_sub_one_iff, le_zero_iff]
 #align fin.sub_one_lt_iff Fin.sub_one_lt_iff
 
+lemma lt_sub_iff {n : ℕ} {a b : Fin n} : a < a - b ↔ a < b := by
```

Comment:
Is `a - b < a ↔ b < a` also true? If so, it would be nice to have both.

Reply:
Unfortunately, not true: `a := 1, b := 1; 0 < 1`  but `1 = 1`.